src/utils/messages_en.ts

```diff
@@ -0,0 +1,46 @@
+export default {
```

Comment:
What the plan for these "messages" files? When we implement the localisation solution, I don't think we want to have two different kinds of strings files hanging around, do we? And will the nested objects work? I think that will make it unnecessarily complicated, don't you? 

(Feel free to reply with just "I have a plan" and we can discuss it later)